Sources/FluentKit/Model/Model.swift

```diff
@@ -227,10 +227,16 @@ extension AnyModel {
     }
 
     var anyID: AnyID {
-        guard let id = Mirror(reflecting: self).descendant("_id") as? AnyID else {
-            fatalError("id property must be declared using @ID")
+        var mirror: Mirror = Mirror(reflecting: self)
+        while true {
+            if let id = mirror.descendant("_id") as? AnyID { return id }
+            if let m = mirror.superclassMirror {
+                mirror = m
+            }
+            else {
```

Comment:
Can you put the elses on the same line as the closing `}`?

Reply:
No problem. The next commit will do this in both places.